PsfLauncher/main.cpp

```diff
@@ -13,11 +13,21 @@
 #include <psf_runtime.h>
 #include <shellapi.h>
 #include <combaseapi.h>
+#include <TraceLoggingProvider.h>
+#include "Telemetry.h"
+
+TRACELOGGING_DECLARE_PROVIDER(g_Log_ETW_ComponentProvider);
+TRACELOGGING_DEFINE_PROVIDER(
+	g_Log_ETW_ComponentProvider,

```

Comment:
nit: The tabs in this project have been using 4 spaces. Should change this throughout to remain consistent.

Reply:
Changed